app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RecipesController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:index, :show]
```

Comment:
People who don't have an account shoulld not be able to see the details for the recipes

Reply:
check! still have to make them want more, true true. so i deleted the "show".